README.md

```diff
@@ -1,6 +1,28 @@
 # AI Multimodal Public Demos
 
-The demos in this repostiory rely solely on public data sets.
+This repository is part of [AIMM project](https://github.com/AI-multimodal): a joint and collaborative effort between scientists from the following Labs:
+- [Argonne National Laboratory](https://www.anl.gov/): Advanced Photon Source (APS) and Center for Nanoscale Materials (CNM)
+- [Brookhaven National Laboratory (BNL)](https://www.bnl.gov/): Center for Functional Nanomaterials (CFN) and National Synchrotron Light Source II (NSLS-II)
+- [Lawrence Berkeley National Laboratory (LBL)](https://www.lbl.gov/): Advanced Light Source (ALS)
+
+The current contributors and authors working in this project are:
```

Comment:
@elistavitski Do you have handy a list of all the post-doc personnel on this project and their affiliations?

Reply:
ANL Postdocs: 
- Inhui Hwang
- Yiming Chen
BNL Scientists
- Matthew Carbone
BNL postdocs
- Zhu Liang 
- Fanchen Meng

Reply:
Is Pandolfi still on the project?

Reply:
Ron Pandolfi asked to step back, if I understood his intentions correctly, so I think it would be appropriate to omit his name.

Reply:
Also, I think we should remove the explicit "Scientists" and "Post-docs" designation. (Not all staff here are best classified as "scientists", like Dylan.) Just list everybody, with staff first.